app/interfaceactiontablehandler.py

```diff
@@ -0,0 +1,386 @@
+import json
+import math
+
+import aiohttp_jinja2
+
+from aiohttp.client_exceptions import (ClientResponseError)
+from aiohttp.web import HTTPFound, RouteTableDef
+from aiohttp_session import get_session
+from structlog import get_logger
+
+from app.searchcriteria import (
+    store_search_criteria,
+    retrieve_job_roles,
+    retrieve_assignment_statuses,
+    clear_stored_search_criteria  
+)
+
+from app.searchfunctions import (
+    get_all_assignment_status,
+    get_employee_records, 
+    allocate_search_ranges,
+    iat_employee_record_table,
+    iat_employee_table_headers,
+    get_distinct_job_role_short,
+    get_employee_count
+)
+
+from . import (NEED_TO_SIGN_IN_MSG, NO_EMPLOYEE_DATA, SERVICE_DOWN_MSG)
+from . import saml
+from .flash import flash
+from flask import Flask
+
+import sys
+import os
+
+
+logger = get_logger('fsdr-ui')
+interface_action_handler_table_routes = RouteTableDef()
+
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
```

Comment:
I'm unsure whether you should be modifying sys.path here. It's generally a bad idea.
What is it used to accomplish?

Reply:
This was copied from indexhandler.py when creating the new file, I assumed it played a role in adding the current file to the path and therefore made it able to be called, on examination the other handlers don't have this code present, so I shall remove it